pkg/golinters/gosec.go

```diff
@@ -149,6 +150,19 @@ func toGosecConfig(settings *config.GoSecSettings) gosec.Config {
 	return conf
 }
 
+func convertScoreToString(score issue.Score) string {
+	switch score {
+	case issue.Low:
+		return "low"
+	case issue.Medium:
+		return "medium"
+	case issue.High:
+		return "high"
+	default:
```

Comment:
enable in project `exhaustive` linter? 🤔

Reply:
It's a false positive producer, I'm sure that is a good idea.

Reply:
not sure* 
? :)